src/containers/Chat/index.tsx

```diff
@@ -151,13 +153,39 @@ export class Component extends React.Component<ComponentProps, {}> {
     }
 
     public async navigationButtonPressed({ buttonId }: any) {
-        if (buttonId === 'delete') {
+        const { conversation, session } = this.props;
+
+        const conversationalists = conversation.participants.filter(
+            (conversationalist) => conversationalist.userId != makeUserId(session.username)
+        );
+
+        const convName = makeUsername(conversationalists[0].userId);
+        const convDisplayName = this.renderName(convName);
+
+        if (buttonId === 'call') {
+            if (this.props.tree.isPending.get()) {
+                return;
+            }
+
             this.props.tree.isPending.set(true);
+
             try {
-                await removeConversation(this.props.tree.removeConversationResponse, this.props.conversation.uuid);
-                await Navigation.pop(this.props.componentId);
-            } finally {
-                this.props.tree.isPending.set(false);
+                const User = {
```

Comment:
Change `makeOutgoingCall` service to accept username and displayName only. Don't construct a User instance, it is wrong because you cannot provide all User fields.